addons/website_slides/models/slide_slide.py

```diff
@@ -372,7 +372,7 @@ def _compute_embed_counts(self):
         for slide in self:
             slide.embed_count = mapped_data.get(slide.id, 0)
 
-    @api.depends('slide_ids.sequence', 'slide_ids.slide_category', 'slide_ids.is_published', 'slide_ids.is_category')
+    @api.depends('slide_ids.sequence', 'slide_ids.slide_category', 'slide_ids.is_published', 'slide_ids.is_category', 'slide_ids.completion_time', 'slide_ids.active')
```

Comment:
I see that you added `slide_ids.active` to the dependencies but I don't see you use it.

Are the archived slides really removed from your computation ? 

Since, I think, if a slide is archived it should not be taken into account in the statistics but, when it is unarchived, then we add to the computation.

Reply:
Just did a recheck on the runbot and I was wrong for this one indeed `slide_ids.active` is necessary for the frontend computation of a section's duration, my bad.